### PR TITLE
[DDW-174] also run js test suite on travis

### PR DESCRIPTION
**Pull request by DominikGuzei**
The target of this PR is to make node / JS test suite run on travis additionally to the current rust tests.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,9 +4,18 @@ rust:
...
   - nightly
-#matrix:
-#  allow_failures:
-#    - rust: nightly
 script:
   - cargo build --verbose --all
   - cargo test --verbose --all
+matrix:
+#  allow_failures:
+#    - rust: nightly
```
Comment (NicolasDP) [inline .travis.yml]: can you remove these comments as well? They are no longer needed anyway. We need to build on nightly.

Comment (DominikGuzei) [reply .travis.yml]: Yeah sure 😉 should i remove the other rust build targets (stable, beta)?

**Comment by DominikGuzei** [comment]
@NicolasDP there are some new javascript tests which are failing (also on my local machine) … is that intentional?

**Comment by NicolasDP** [comment]
yes, we were creating addresses with a single key distribution, which is not the current scheme. This has been fixed but not the test vector at the JS level. I will push something soon about it

**Comment by DominikGuzei** [comment]
@NicolasDP ok, this PR is done though - the tests are running correctly for the rust `nightly` build. Please review and merge!

File: .travis.yml [hunk 2]
```diff
@@ -1,12 +1,17 @@
...
-#  allow_failures:
-#    - rust: nightly
 script:
   - cargo build --verbose --all
   - cargo test --verbose --all
+matrix:
+  include:
+    - rust: stable
+    - rust: beta
+    - rust: nightly
```
Comment (NicolasDP) [inline .travis.yml]: duplicating this line and adding a env variable `TEST_NODEJS_BINDINGS` would be enough and would show in the travis-ci board

**Comment by DominikGuzei** [comment]
@NicolasDP can you make the necessary changes - im on holidays until 4th of April!

**Comment by NicolasDP** [comment]
I had to do a rebase of the code to master in order to fix the travis script.

**Pull request merged**